pandas/io/gbq.py

```diff
@@ -30,10 +30,10 @@ def _try_import() -> ModuleType:
 
 
 def read_gbq(
-    query: str,
+    query_or_table: str,
     project_id: str | None = None,
     index_col: str | None = None,
-    col_order: list[str] | None = None,
+    columns: list[str] | None = None,
```

Comment:
Likewise, let's not change the order of parameters since they could be used either positionally or as keyword arguments.